.github/scripts/filter_test_configs.py

```diff
@@ -497,7 +485,7 @@ def perform_misc_tasks(
 
     # Obviously, if the job name includes unstable, then this is an unstable job
     is_unstable = job_name and IssueType.UNSTABLE.value in job_name
-    if not is_unstable and test_matrix:
+    if not is_unstable and test_matrix and test_matrix.get("include"):
```

Comment:
Do you mind what the mps bug was?  I don't understand how this fixes it

Reply:
I should have added a link to the description, but here it is https://github.com/pytorch/pytorch/actions/runs/8329872641/job/22802012305?pr=121381#step:9:151.  So you can see the issue there in `Is the current job unstable? True`, this is wrong because there is no unstable issue for MPS.  The bug is in this part `is_unstable = all(IssueType.UNSTABLE.value in r for r in test_matrix["include"])` which returns true when `test_matrix["include"]` is an empty array :)

Reply:
haha true by vacuity